aries-site/src/examples/components/header/HeaderAvatarExample.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Avatar, Box, Button, Header, Text } from 'grommet';
+import { Hpe } from 'grommet-icons';
+
+export const HeaderAvatarExample = () => {
+  return (
+    <Header fill="horizontal">
+      <Button>
+        <Box direction="row" align="center" gap="medium">
+          <Hpe color="brand" />
+          <Box direction="row" gap="xsmall">
+            <Text weight="bold">HPE</Text>
+            <Text>App Name</Text>
+          </Box>
+        </Box>
+      </Button>
+      <Avatar background="background-contrast">ES</Avatar>
```

Comment:
I know in the directions you had put to wrap 'Avatar' in Button so curious why you didnt wrap it here maybe we can have it commented out ? That way if they skip the directions and go straight to copy and paste code they wont miss it

Reply:
Good point, having it interactive here would probably be helpful and more realistic.